net-nds/openldap/openldap-2.4.44.ebuild

```diff
@@ -731,15 +757,15 @@ multilib_src_install() {
 
 		einfo "Installing contrib modules"
 		cd "${S}/contrib/slapd-modules" || die
-		for l in */*.la; do
+		for l in $(find -type f -name "*.la"); do
```

Comment:
Possible... in fact, I haven't consider this question. I just transpose that already exist to avoid potential regression.